templates/kernel/real-time/contact-us.html

```diff
@@ -487,165 +300,447 @@ <h2 class="p-heading--3">How are you currently meeting your latency requirements
                 </label>
               </div>
             </div>
-          </fieldset>
-          <fieldset class="u-no-margin--bottom">
-            <label class="p-heading--3">What advice are you looking for?</label>
+          </div>
+        </fieldset>
+      </div>
+
+      <div class="p-section">
+        <hr class="is-fixed-width" />
+        <fieldset class="p-fieldset-section" id="kind-of-device" aria-required="true">
+          <div class="row--50-50">
+            <div class="col">
+              <legend class="p-heading--2 is-required">What kind of device are you using?</legend>
+            </div>
+            <div class="col">
+              <label class="p-checkbox">
+                <input required
```

Comment:
```suggestion
                <input
```
This is making that individual checkbox required. But there nature, checkboxes cannot be made to be required. You would have to check it with some custom JS